cities/schemas.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str | None = None
+
+
+class CityCreate(CityBase):
+    pass
+
+
+class CityUpdate(BaseModel):
+    additional_info: str
+
+
+class City(CityBase):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class TemperatureBase(BaseModel):
+    date_time: datetime
+    temperature: float
+
+
+class TemperatureCreate(TemperatureBase):
+    city_id: int
+
+
+class Temperature(TemperatureBase):
+    id: int
+    city: City
+
+    class Config:
+        from_attributes = True
```

Comment:
Similar to the `City` model, ensure that you use `orm_mode = True` if you want to enable ORM serialization for the `Temperature` model.